purchase_deposit/wizard/purchase_make_invoice_advance.py

```diff
@@ -169,6 +169,23 @@ def create_invoices(self):
             amount = self.amount
             if self.advance_payment_method == "percentage":  # Case percent
                 amount = self.amount / 100 * order.amount_untaxed
+            # calculate all deposit lines
+            sum_deposit = (
+                sum(
+                    line.price_unit
+                    for line in order.order_line
+                    if (line.is_deposit and line.invoice_lines)
+                )
+                or 0.00
+            )
+            if (sum_deposit + amount) > order.amount_total:
```

Comment:
```suggestion
            if (sum_deposit + amount) > order.amount_total:
```

Use `float_compare` instead.